bundles/binding/org.openhab.binding.sapp/src/main/java/org/openhab/binding/sapp/SappUpdatePendingRequests.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.sapp;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Storage for items changed in cofiguration, to be reloaded. Synchronized access 
+ * 
+ * @author Paolo Denti
+ * @since 1.8.0
+ */
+public class SappUpdatePendingRequests {
+	
+	private Boolean mutex;
```

Comment:
The `mutex` member is not needed.  You could instead do
```
private Set<String> pendingUpdateRequests = Collections.synchronizedSet(new HashSet<String>());
```
and use it without `synchronized` blocks for the single-line methods, and have no constructor for the class.